pep-0676.rst

```diff
@@ -9,20 +9,22 @@ Content-Type: text/x-rst
 Created: 01-Nov-2021
 Post-History: 23-Sep-2021
 
+
 Abstract
 ========
 
 This PEP addresses the infrastructure around rendering PEP files from
-reStructuredText_ files to HTML webpages. We aim to specify a self-contained
+`reStructuredText`_ files to HTML web pages. We aim to specify a self-contained
```

Comment:
I'd probably stick with "webpage", over time words in English tend to become joined with use, and this is happening with "web page" -> "webpage".

See https://english.stackexchange.com/a/540931/9001 and https://books.google.com/ngrams/graph?content=web+page%2Cweb+pages%2Cwebpage%2Cwebpages&year_start=1980&year_end=2019&corpus=26&smoothing=3 for an updated chart:

![image](https://user-images.githubusercontent.com/1324225/146676120-0d3ec303-7fd3-41f6-b671-9f82ec60e09e.png)


(There's currently 3 vs. 3 of each in PEPs, so no precedent here.)


```suggestion
`reStructuredText`_ files to HTML webpages. We aim to specify a self-contained
```

Reply:
Yeah, makes sense. May as well go with the more modern usage; I almost didn't change it (and should have thought better of it).